CI/travis/before_install_linux

```diff
@@ -19,21 +19,21 @@ handle_centos() {
 	yum info epel-release
 	yum -y install epel-release
 
+	if is_centos_at_least_ver "8" ; then
```

Comment:
this doesn't work before `redhat-lsb-core` is installed;
it requires `lsb_release` installed;

though; maybe it would be a better idea to rework those distro functions (i.e. `get_codename()`, `get_dist_id()` & `get_version()`) to just manually parse `/etc/os-release`;
no idea;
i liked using `lsb_release` since the output format is simple/unique, and doesn't need extra parsing, but given the hassle of installing it on bare-bone docker images, it may not be worth it;

Reply:
Yeah, it was a last try before I went to bed last night... I was hoping it would make some of the logic easier - I will flip things back.